src/main/scala/be/doeraene/sjsreflect/Reflect.scala

```diff
@@ -23,6 +21,8 @@ object Reflect {
 
   private[Reflect] def makeClassesByName(): js.Dictionary[Class[_]] = stub()
 
+  private[Reflect] def makeClassArray(): Array[Class[_]] = stub()
```

Comment:
This one should still be a `js.Array`, otherwise the `JSArrayConstr` used in the linker plugin does not comply to. That's why the build failed.

(The other one, the public method, should return `Array[Class[_]]` as discussed before.)